lib/response.js

```diff
@@ -28,24 +38,18 @@ Response = (function () {
   Response.prototype.job_update = function (status, data) {
     this.meta('set_job_status', 'in_progress');
     this.meta('set_job_status_message', status);
-    if(data) {
-      this._respond(data || {});
-    }
+    this._respond(data || {});
   };
   Response.prototype.job_complete = function (status, data) {
     this.meta('set_job_status', 'done');
     this.meta('set_job_status_message', status);
-    if(data) {
-      this._respond(data || {});
-    }
+    this._respond(data || {});
   };
   Response.prototype.job_fail = function (status, msg, data) {
```

Comment:
Consider what to do in the case where we want a route to return specific html content and also update a job status.